#! /bin/sh /usr/share/dpatch/dpatch-run
## 53_fix_pkgconfig_libs.dpatch by Daniel T Chen <crimsun@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: alsa.pc's Libs shouldn't contain -lm -ldl -lpthread
## DP: http://hg-mirror.alsa-project.org/alsa-lib?cs=7bc021fb4bf1;style=raw

@DPATCH@
diff -urNad alsa-lib-1.0.13~/utils/alsa.pc.in alsa-lib-1.0.13/utils/alsa.pc.in
--- alsa-lib-1.0.13~/utils/alsa.pc.in	2006-09-29 07:42:58.000000000 -0400
+++ alsa-lib-1.0.13/utils/alsa.pc.in	2007-02-04 18:47:19.000000000 -0500
@@ -7,7 +7,8 @@
 Description: Advanced Linux Sound Architecture (ALSA) - Library
 Version: @VERSION@
 Requires: 
-Libs: -L${libdir} -lasound -lm -ldl -lpthread
+Libs: -L${libdir} -lasound
+Libs.private: -lm -ldl -lpthread
 # -I${includedir}/alsa below is just for backward compatibility
 # (it was set so mistakely in the older version)
 Cflags: -I${includedir} -I${includedir}/alsa
